tests/manual-test-cases/Group21-Registries/21-1-Whitelist.robot

```diff
@@ -93,3 +93,38 @@ Basic Whitelisting
     Should Contain  ${output}  Access denied to unauthorized registry
 
     Cleanup VIC Appliance On Test Server
+
+Configure Registry CA
+    # Install VCH without registry CA
+    Remove Environment Variable  GOVC_USERNAME
+    Remove Environment Variable  GOVC_PASSWORD
```

Comment:
Seem to remember the test above failing for one reason or another without doing this